engine/src/main/java/org/terasology/rendering/nui/layouts/CardLayout.java

```diff
@@ -26,20 +26,43 @@
 import java.util.Iterator;
 import java.util.Map;
 
+/**
+ * A layout that allows for a single widget to be displayed among a list of stored widgets
+ */
 public class CardLayout extends CoreLayout<LayoutHint> {
+    /**
+     * The id of the currently displayed widget
+     */
```

Comment:
Generally javadoccing private stuff isn't mandated, but it can still be a nice thing to do for stuff like the `CTRL-Q` shortcut. So there's really no harm in doing it with javadoc rather than a regular comment.

I think the consistency angle is that if you have 5 public 5 protected and 5 private fields on a class you might as well describe them all via Javadoc rather than switch to regular comments for the private ones :-)